manifests/charts/istio-control/istio-discovery/templates/NOTES.txt

```diff
@@ -11,3 +11,9 @@ Next steps:
     * https://istio.io/latest/docs/tasks/security/
     * https://istio.io/latest/docs/tasks/policy-enforcement/
     * https://istio.io/latest/docs/tasks/policy-enforcement/
+  * Review the list of actively supported releases, CVE publications and our hardening guide:
```

Comment:
Seems a bit odd to tell them to review the supported versions *after* they installed.